www/src/assets/logos.js

```diff
@@ -5,6 +5,7 @@ import ReactJSIcon from "./react.svg"
 import SegmentIcon from "./segment.svg"
 import FormidableIcon from "./formidable.svg"
 import FabricIcon from "./fabric.svg"
+import NetlifyIcon from "./netlify.svg"
```

Comment:
Completely unrelated to this PR, but we should look into making these React components (where SVG is valid) instead of this!

cc @fk